documentation/Color system.md

```diff
@@ -242,6 +242,7 @@ Used to decorate elements where color does convey a specific meaning in componen
 | ----------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
 | `--p-border-radius-base`                  | `0.4rem`                                                                                                     |
 | `--p-border-radius-wide`                  | `0.8rem`                                                                                                     |
+| `--p-border-radius-full`                  | `50%`                                                                                                        |
```

Comment:
This is useful for icon only buttons
![image](https://screenshot.click/Page_Component__Figma_2020-09-24_11-27-20.png)